CHANGES.md

```diff
@@ -57,6 +57,8 @@ To be released.
 ### Added APIs
 
  -  Added `IActonEvaluator` interface.  [[#3082]]
+ -  Added `SwarmOptions.ResourceRegulationOptions` property.  [[#3109]]
```

Comment:
How about `SwarmOptions.TaskRegulationOptions`? (or, is there any plan for applying this option other than task counts?)